### ISSUE TITLE
Problemas no Microsoft Azure PowerShell

### ISSUE THREAD

**Issue by JackHackham** [PowerShell/PSReadLine#2014]
Boa tarde, 

Fui tentar iniciar uma sessão no shell do Microsoft Azure e ao digitar já me ocorreu um erro e o mesmo foi informado para postar este problema aqui, segue o problema abaixo:

```
Opa, algo deu errado. Relate este bug com os detalhes abaixo.
Relate no GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Últimas 2 Chaves:
 l l

Exceção:
System.IO.IOException: Parâmetro incorreto.

   em System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   em System.IO.__Error.WinIOError()
   em System.Console.set_CursorVisible(Boolean value)
   em Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorVisible(Boolean value)
   em Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   em Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   em Microsoft.PowerShell.PSConsoleReadLine.Render()
   em Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   em Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   em Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   em Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   em Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)"
-----------------------------------------------------------------------
```

Imagem do código

![image](https://user-images.githubusercontent.com/75494921/101192040-3f6bd780-3639-11eb-9069-f7e8dd09ba48.png)

Podem me ajudar por gentileza?

Obrigado

**Comment by iSazonov** [comment]
@JackHackham Please translate your report to English.

**Comment by JackHackham** [comment]
Good afternoon,

When trying to start a session in the Microsoft Azure shell, typing an error occurs and it was provided to post the problem here on Github, the problem is as follows:

`
Oops, something went wrong. Report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 2 Keys:
 l l


Exception:
System.IO.IOException: Incorrect parameter.

   em System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   em System.IO.__Error.WinIOError()
   em System.Console.set_CursorVisible(Boolean value)
   em Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorVisible(Boolean value)
   em Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   em Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   em Microsoft.PowerShell.PSConsoleReadLine.Render()
   em Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   em Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   em Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   em Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   em Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)"
-----------------------------------------------------------------------`

Pode me ajudar por gentileza?

Obrigado

**Comment by daxian-dbw** [comment]
This issue was already fixed (see #1306). Please upgrade to the [2.1.0 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.1.0) from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.
